src/Centreon/Domain/PlatformTopology/PlatformTopologyService.php

```diff
@@ -79,6 +80,9 @@ class PlatformTopologyService implements PlatformTopologyServiceInterface
      */
     private $monitoringServerService;
 
+    // Version of the API used to register the platform on the central
+    public const API_VERSION = 'v2.0';
```

Comment:
I prefered to add this constant instead of hardcoding it in the payload used to send the API request to the n-1 platform

Reply:
Adding a reference to the API context is forbidden in the business context.